### PR TITLE
fix: Percentile stats for bfb results

### PR DESCRIPTION
**Pull request by KShivendu**
This PR should:
- Remove the p99 values for RPS metric 
- Introduce median (p50) for all the metrics

### All Submissions:

* [x] Contributions should target the `dev` branch. Did you create your branch from `dev`?
* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you formatted your code locally using `cargo +nightly fmt --all` command prior to submission?
3. [x] Have you checked your code using `cargo clippy --all --all-features` command?

### CODE REVIEW

**Review by timvisee — APPROVED** [review]
Nice!

File: src/main.rs [hunk 1]
```diff
@@ -347,28 +347,33 @@ fn write_to_json(path: &String, results: SearcherResults) {
...
 
-    let avg_time: f64 = timings.iter().sum::<f64>() / timings.len() as f64;
-    let min_time: f64 = timings.first().copied().unwrap_or(0.0);
-    let max_time: f64 = timings.last().copied().unwrap_or(0.0);
-    let p95_time: f64 = timings[(timings.len() as f32 * 0.95) as usize];
+    let avg_time: f64 = values.iter().sum::<f64>() / values.len() as f64;
+    let min_time: f64 = values.first().copied().unwrap_or(0.0);
+    let max_time: f64 = values.last().copied().unwrap_or(0.0);
+    let p50_time: f64 = values[(values.len() as f32 * 0.50) as usize];
+    let p95_time: f64 = values[(values.len() as f32 * 0.95) as usize];
```
Comment (agourlay) [inline src/main.rs:361]: nitpick you can now push down `p95_time` under `show_percentiles`

File: src/main.rs [hunk 2]
```diff
@@ -347,28 +347,33 @@ fn write_to_json(path: &String, results: SearcherResults) {
...
+    values.sort_unstable_by(|a, b| a.partial_cmp(b).unwrap());
 
-    let avg_time: f64 = timings.iter().sum::<f64>() / timings.len() as f64;
-    let min_time: f64 = timings.first().copied().unwrap_or(0.0);
-    let max_time: f64 = timings.last().copied().unwrap_or(0.0);
-    let p95_time: f64 = timings[(timings.len() as f32 * 0.95) as usize];
+    let avg_time: f64 = values.iter().sum::<f64>() / values.len() as f64;
+    let min_time: f64 = values.first().copied().unwrap_or(0.0);
+    let max_time: f64 = values.last().copied().unwrap_or(0.0);
+    let p50_time: f64 = values[(values.len() as f32 * 0.50) as usize];
```
Comment (agourlay) [inline src/main.rs:360]: where is `p50_time` used?

Comment (agourlay) [reply src/main.rs:360]: it is `median`,, sorry for the noise

File: src/main.rs (hunk 1 above)
Comment (KShivendu) [reply src/main.rs:361]: Sure. I think I did this but messed up my commits and then it was gone. Thanks!

**Comment by KShivendu** [comment]
Thanks @agourlay @timvisee. Merging this now.

**Pull request merged**